en/samples/easy-crm.md

```diff
@@ -16,13 +16,15 @@ Visit [easycrm.samples.commercial.abp.io](http://easycrm.samples.commercial.abp.
 
 When you download and open the zip file, you will see two folders:
 
-* **mvc** folder contains the server side, the MVC (Razor Pages) UI and the Blazor UI.
+* **aspnet-core** folder contains the server side, the MVC (Razor Pages) UI and the Blazor UI.
 * **angular** folder contains the Angular UI.
 
 ### Server Side / MVC (Razor Pages) Application
 
-* Open the solution (inside the mvc folder) in **Visual Studio 2019** or later (or with another IDE that supports ASP.NET Core).
-* This project use `Sqlite`, the default database folder is located at appsettings (`"SqliteDbFolder": "c:\\Temp\\SqliteDbs\\"`, you should create this folder).
+* Open the solution (inside the aspnet-core folder) in **Visual Studio 2019** or later (or with another IDE that supports ASP.NET Core).
+* This project use `Sqlite`, the default database folder is located at appsettings (`"SqliteDbFolder": "sqliteDbs"`, this folder is located in the MVC project).
+* Open the `appsettings.json` file in the `Volo.EasyCrm.Web` application and set `"UseDynamicDatabase": "false"`.
+    > The MVC project is creating new database for each unique visitor. And the visitor id is stored at cookies. When you set `UseDynamicDatabase` as a `true`, you cannot run Blazor & Angular projects because they have no cookie implementation. Be aware it is set as `false` for running Blazor & Angular applications.
```

Comment:
This is a bad developer experience.
`UseDynamicDatabase` should be `false` by default. We can override the setting via environment variables in our demo.

Reply:
Right, I will update it. 👍